hoomd/hpmc/test-py/enthalpic_interaction.py

```diff
@@ -198,5 +198,42 @@ def tearDown(self):
             del self.log;
             context.initialize();
 
+# These tests check the methods for functionality
+class patch_test_alpha_methods(unittest.TestCase):
+
+    def setUp(self):
+        self.dummy_potential = """ return 0;
+                               """
+        system = init.create_lattice(unitcell=lattice.sc(a=2), n=2);
+        self.mc = hpmc.integrate.sphere(seed=1,d=0.1);
+        self.mc.shape_param.set('A',diameter=0);
+
+    def assert_patch(self, patch):
+
+        # check alpha list is set properly
+        patch.alpha = [-1., 2.7, 10]
+        np.testing.assert_almost_equal([-1., 2.7, 10], patch.alpha, decimal=6)
```

Comment:
- [ ] Use of `assert_almost_equal` is discouraged by the numpy documentation, please use `assert_allclose`